doc/orchestrator.rst

```diff
@@ -1,171 +1,627 @@
+    requires that the ``Model`` finish before the experiment continues.
+    This is because we will request tensors from the database that
+    are inputted by the ``Model`` we launched.
+
+Poll Data Using Clients
+-----------------------
+Next, check if the tensor exists in the standard database using ``Client.poll_tensor()``.
+This function queries for data in the database. The function requires the tensor name (`name`),
+how many milliseconds to wait in between queries (`poll_frequency_ms`),
+and the total number of times to query (`num_tries`). Check if the data exists in the database by
+polling every 100 milliseconds until 10 attempts have completed:
 
-  colo_settings = exp.create_run_settings(exe="./some_mpi_app")
+.. code-block:: python
+
+  # Retrieve the tensors placed by the Model
+  value_2 = driver_client_standard_db.poll_key("tensor_2", 100, 10)
+  # Validate that the tensor exists
+  logger.info(f"The tensor is {value_2}")
+
+When you execute the driver script, the output will be as follows::
+
+  23:45:46 osprey.us.cray.com SmartSim[87400] INFO The tensor is True
+
+Cleanup
+-------
+Finally, use the ``Experiment.stop()`` function to stop the database instances. Print the
+workflow summary with ``Experiment.summary()``:
+
+.. code-block:: python
 
-  colo_model = exp.create_model("colocated_model", colo_settings)
-  colo_model.colocate_db_uds(
-          db_cpus=1,              # cpus given to the database on each node
-          debug=False             # include debug information (will be slower)
-          ifname=network_interface # specify network interface(s) to use (i.e. "ib0" or ["ib0", "lo"])
-  )
-  exp.start(colo_model)
+  # Cleanup the database
+  exp.stop(standard_db)
+  logger.info(exp.summary())
 
+When you run the experiment, the following output will appear::
 
-By default, SmartSim will pin the database to the first _N_ CPUs according to ``db_cpus``. By
-specifying the optional argument ``custom_pinning``, an alternative pinning can be specified
-by sending in a list of CPU ids (e.g [0,2,range(5,8)]). For optimal performance, most users
-will want to also modify the RunSettings for the model to pin their application to cores not
-occupied by the database.
+  |    | Name           | Entity-Type   | JobID       | RunID   | Time    | Status    | Returncode   |
+  |----|----------------|---------------|-------------|---------|---------|-----------|--------------|
+  | 0  | model          | Model         | 1658679.3   | 0       | 1.3342  | Completed | 0            |
+  | 1  | orchestrator_0 | DBNode        | 1658679.2+2 | 0       | 42.8742 | Cancelled | 0            |
 
-.. warning::
+.. _colocated_orch_doc:
+====================
+Colocated Deployment
+====================
+--------
+Overview
+--------
+During colocated ``Orchestrator`` deployment, a SmartSim ``Orchestrator`` (the database) runs on
+the ``Models`` compute node(s). A colocated ``Orchestrator`` can only be deployed on a single node
```

Comment:
A colocated ``Orchestrator`` can only be deployed on a single node
and cannot be sharded (distributed) over multiple nodes.

->

A colocated ``Orchestrator`` can only be deployed as isolated instances on each compute node
and cannot be clustered over multiple nodes.